recipes/cmake/3.x.x/conanfile.py

```diff
@@ -1,20 +1,24 @@
-import os
 from conan import ConanFile
+from conan.tools.files import chdir, copy, rmdir, get, files
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.gnu import Autotools, AutotoolsToolchain
```

Comment:
No AutotoolsDeps? Is it robust?

Reply:
Could you please elaborate, not an autotools users so I am not sure what you might be referring too?

Reply:
openssl is a dependency, AutotoolsDeps injects dependencies informations in compile and link flags of autotools, so that it can work out of the box. Usually if upstram configure.ac relies on pkgconf to find & inject these flags, AutotoolsDeps is not needed, otherwise it may be fragile and AutotoolsDeps is here to improve robustness.